admin/frontend/views/pages/list.pug

```diff
@@ -27,7 +27,12 @@ block content
             thead
               tr
                 for prop in currentResource.decorate().getListProperties()
-                  th.text-small= h._.startCase(prop.name())
+                  th(class= { 'main' : h.isMainColumn(prop.name())})
+                    - isSortedByCurrentPropName = sort.sortBy === prop.name()
+                    a(href=h.listUrl(currentResource, { sort: { sortBy: prop.name(), sortDirection: sort.direction === 'asc' && isSortedByCurrentPropName ? 'desc' : 'asc' }}))
```

Comment:
[OPT] sort direction condition could be another variable to increase readability